lib/fastlane/plugin/firebase_test_lab/helper/ios_validator.rb

```diff
@@ -14,14 +14,22 @@ def self.validate_ios_app(file_path)
             end
 
             conf = Plist.parse_xml(xctestrun_files.first.get_input_stream)
-            size = conf.size
-            if conf['__xctestrun_metadata__']
-              size -= 1
-            end  
-            unless size == 1
-              UI.user_error!("The app bundle may contain only one scheme, #{size} found")
+
+            # Grab the first non-'__xctestrun_metadata__' scheme
+            scheme_conf = nil
+            conf.each do |key, value|
+              if scheme_conf.nil?
+                unless key == '__xctestrun_metadata__'
```

Comment:
This "unless" needs to be outside of the if block. If you got an input:
ARandomScheme
__xctestrun_metadata__

It will cause the one-scheme-only error.

Reply:
Fair comment.

I've re-introduced the size check and fixed this up.